lib/src/components/slider/slider.dart

```diff
@@ -113,25 +117,49 @@ class _SliderPainter extends CustomPainter {
       ..style = PaintingStyle.stroke
       ..strokeWidth = 1;
 
-    final RRect track = RRect.fromLTRBR(0.0, 12.0, size.width, size.height - 12, const Radius.circular(12.0));
-    final RRect active = RRect.fromLTRBR(
-      0.0,
-      12.0,
-      (size.width * value) > 24 ? (size.width * value) : 24.0,
-      size.height - 12.0,
-      const Radius.circular(12.0),
-    );
+    final Paint dividerPaint = Paint()
+      ..color = outlineColor
```

Comment:
outlineColor for the dividers doesn't make a lot of sense, could you rename this to something better? and if it's used by something else, maybe look into decoupling to allow customizability?

Reply:
Oh yeah, seems like I forgot to change that